core.go

```diff
@@ -147,54 +65,12 @@ type (
 		DeviceClass string `json:"deviceClass"`
 	}
 
-	// DeviceCommand represents a command received from the MODE cloud.
```

Comment:
I have moved them into mqtt_mode.go, so it isn't totally gone. But, your point is good about key-value's. I will improve the handling in this library. What do you think about moving the cache into a common applications library?

Reply:
I agree about moving the cache into a common applications library, in other word, a high level SDK.
I think the high level SDK also should be public. Because many developer will want to use the feature.

Reply:
In my opinion, that's fine. There is no secret algorithms and only our customers are using this code. I'll discuss with Ethan.

Reply:
I see. That's fine.